ci-operator/step-registry/openshift/e2e/libvirt/conf/openshift-e2e-libvirt-conf-commands.sh

```diff
@@ -323,6 +323,130 @@ EOF
     fi
 fi
 
+# Skip the following tests for clusters with multi-architecture compute nodes
+#
+# The list of skipped tests is the result of the following regular expression against the
+# list of all available tests for provider_none and suite openshift/conformance/parallel
+#
+# TEST_SKIPS: deploymentconfigs\| should expose cluster services outside the cluster\|
+#        FIPS TestFIPS\| Multi-stage image builds should succeed\| Optimized image
+#        builds should succeed\| build can reference a cluster service\| custom build
+#        with buildah\| oc new-app should succeed\| prune builds based on settings\|
+#        s2i build with a root\| verify /run filesystem contents\| oc can run\| oc
+#        debug\| oc idle\| Pods cannot access\| Image append should create\| Image
+#        extract should extract\| Image info should display\| Image layer subresource\|
+#        oc tag should change image\| when installed on the cluster should\| OpenShift
+#        alerting rules\| The HAProxy router should\| egressrouter cni resources\|
+#        pod should start\| pod sysctls\| build volumes should mount given secrets
+#        and configmaps into the build pod
+HETEROGENEOUS_CLUSTER_TYPE_REGEX="^(libvirt-s390x-amd64)|(libvirt-amd64-s390x)$"
+if [ "${TEST_TYPE}" == "conformance-parallel" ] && [[ "${CLUSTER_TYPE}" =~ $HETEROGENEOUS_CLUSTER_TYPE_REGEX ]]; then
```

Comment:
Nitpick: can we maintain variable reference consistency here?  Thanks!
`${HETEROGENEOUS_CLUSTER_TYPE_REGEX}`

Reply:
Done. Good that you pointed me to that place again, regex was also slightly wrong (`"^(libvirt-s390x-amd64)|(libvirt-amd64-s390x)$"` should be `"^(libvirt-s390x-amd64|libvirt-amd64-s390x)$"`).